### ISSUE TITLE
Some question about iou and beta

### ISSUE THREAD

**Issue by wangjue-wzq** [hkzhang95/DynamicRCNN#10]
Hello! I am interested in your paper. Dynamic RCNN obtains iou and beta according to the current detection results. Have you directly set iou and beta to the values calculated by dynamic rcnn, such as 0.6 and 0.2, for training?

**Comment by hkzhang95** [comment]
No, but we have tried some values towards the observed trend.

As shown in Table 5, we try some common values of beta (e.g. 0.5-2.0) and find that a smaller value is better than the default one. I also remember directly using L1 Loss instead of SmoothL1 Loss is helpful, but not as good as our DSL. I believe that raising the IoU threshold will draw a similar conclusion.

In fact, as analyzed in the paper, we do not think that directly setting a fixed threshold or hyperparameter is the optimal solution. We hope our work can inspire further researchers to rethink the framework in a dynamic way.

**Comment by wangjue-wzq** [comment]
Thank you for your reply! I used dynamic rcnn in the custom data set, but it didn't work. Iou approaches 0.6 after 5 epochs, and beta approaches 0.2 after 2 epochs. It is worth noting that 60% of the object scale in this data set is smaller than 32×32 pixels. Can you give me some suggestions?

**Comment by hkzhang95** [comment]
The reason is not clear according to your description, but I have some guesses.

- As for the object scale part, it seems like the default anchor setting is not reasonable for your custom dataset. You can change the anchor scales and ratios, then calculate the recall of current GTs to decide whether the anchor setting is ok.
- As for the Dynamic R-CNN part, our design mainly focus on high quality object detection, so if you use low quality evaluation metrics (like AP50 only), our default hyperparameters may not be helpful.

**Comment by wangjue-wzq** [comment]
Thanks for your suggestion.
